README.md

```diff
@@ -178,8 +178,7 @@ When triggering ad hoc runs in your dev/sandbox environment, or when running the
 
 ### Assets
 
-In each pipeline folder, there is an `assets` directory (`pipelines/pipelines/<xgboost|tensorflow>/<training|prediction>/assets/`). 
-This can be used for any additional files that may be needed during execution of the pipelines. 
+Pipeline folder, contains `assets` directory (`pipelines/assets/`). This can be used for any additional files that may be needed during execution of the pipelines. 
```

Comment:
```suggestion
The folder `pipelines/assets/` can be used for any additional files that may be needed during execution of the pipelines. 
Most importantly this can include your training scripts.
```